project_stage/models/project_stage.py

```diff
@@ -0,0 +1,34 @@
+##############################################################################
+# For copyright and license notices, see __manifest__.py file in module root
+# directory
+##############################################################################
+from odoo import models, fields
+
+
+class ProjectStage(models.Model):
+
+    _name = 'project.stage'
+    _description = 'Project Stage'
+    _order = 'sequence'
+
+    name = fields.Char(
+        string='Stage Name',
+        required=True,
+        translate=True,
+    )
+    description = fields.Text(
+    )
+    sequence = fields.Integer(
+    )
+    case_default = fields.Boolean(
```

Comment:
This field doesn't seem to have any effect on anything. I suggest we drop it.

Reply:
Yeah, seems like it.